lib/matplotlib/colorbar.py

```diff
@@ -915,150 +904,34 @@ def _process_values(self, b=None):
-
-            if isinstance(self.norm, (colors.PowerNorm, colors.LogNorm)):
-                # If using a lognorm or powernorm, ensure extensions don't
-                # go negative
-                if self._extend_lower():
-                    b[0] = 0.9 * b[0]
-                if self._extend_upper():
-                    b[-1] = 1.1 * b[-1]
-            else:
-                if self._extend_lower():
-                    b[0] = b[0] - 1
-                if self._extend_upper():
-                    b[-1] = b[-1] + 1
-        self._process_values(b)
-
-    def _get_extension_lengths(self, frac, automin, automax, default=0.05):
-        """
-        Return the lengths of colorbar extensions.
-
-        This is a helper method for _uniform_y and _proportional_y.
-        """
-        # Set the default value.
-        extendlength = np.array([default, default])
-        if isinstance(frac, str):
-            _api.check_in_list(['auto'], extendfrac=frac.lower())
-            # Use the provided values when 'auto' is required.
-            extendlength[:] = [automin, automax]
-        elif frac is not None:
-            try:
-                # Try to set min and max extension fractions directly.
-                extendlength[:] = frac
-                # If frac is a sequence containing None then NaN may
-                # be encountered. This is an error.
-                if np.isnan(extendlength).any():
-                    raise ValueError()
-            except (TypeError, ValueError) as err:
-                # Raise an error on encountering an invalid value for frac.
-                raise ValueError('invalid value for extendfrac') from err
-        return extendlength
-
-    def _uniform_y(self, N):
-        """
-        Return colorbar data coordinates for *N* uniformly
-        spaced boundaries, plus ends if required.
-        """
-        if self.extend == 'neither':
-            y = np.linspace(0, 1, N)
-        else:
-            automin = automax = 1. / (N - 1.)
-            extendlength = self._get_extension_lengths(self.extendfrac,
-                                                       automin, automax,
-                                                       default=0.05)
-            if self.extend == 'both':
-                y = np.zeros(N + 2, 'd')
-                y[0] = 0. - extendlength[0]
-                y[-1] = 1. + extendlength[1]
-            elif self.extend == 'min':
-                y = np.zeros(N + 1, 'd')
-                y[0] = 0. - extendlength[0]
-            else:
-                y = np.zeros(N + 1, 'd')
-                y[-1] = 1. + extendlength[1]
-            y[self._inside] = np.linspace(0, 1, N)
-        return y
 
-    def _proportional_y(self):
-        """
-        Return colorbar data coordinates for the boundaries of
-        a proportional colorbar.
-        """
+        # otherwise values are set from the boundaries (and probably aren't
```

Comment:
That parenthetical comment is unclear.  Do you mean they aren't actually used?

Reply:
removed comment....